tagreader/utils.py

```diff
@@ -75,14 +75,14 @@ def convert_to_pydatetime(date_stamp: Union[datetime, str, pd.Timestamp]) -> dat
         return date_stamp.to_pydatetime()
     else:
         try:
-            return pd.to_datetime(str(date_stamp), format="ISO8601").to_pydatetime()
+            return pd.to_datetime(date_stamp).to_pydatetime()
```

Comment:
What is the motivation for this change? I think it's best to not do this because the existing code is more forgiving with regards to input format.